app/lib/index.js

```diff
@@ -126,28 +126,28 @@ var migration_functions = {
         page: require('./page.js'),
         resource:  require('./resource.js'),
         role: undefined,
-        tag: undefined,
+        tag: require('./tag.js'),
         assign:     require('./assign.js'),
         assignment: require('./assignment.js'),
         chat: require('./chat.js'),
         feedback: require('./feedback.js'),
         glossary: require('./glossary.js'),
         scorm:  require('./scorm.js'),
         wiki: undefined,
-        workshop: undefined,
-        discussion: undefined,
-        book: undefined,
-        folder: undefined,
-        imscp: undefined,
+        workshop: require('./workshop.js'),
+        discussion: require('./discussion.js'),
+        book: require('./book.js'),
+        folder: require('./folder.js'),
+        imscp: require('./imscp.js'),
         label: undefined,
-        url: undefined,
+        url: require('./url.js'),
         quiz: require('./quiz.js'),
-        choice: undefined,
+        choice: require('./choice.js'),
         lesson: require('./lesson.js'),
-        lti: undefined,
-        data: undefined,
-        journal: undefined,
-        bigbluebuttonbn: undefined
+        lti: require('./lti.js'),
+        data: require('./data.js'),
+        journal: "Can't be migrated because mdl_journal table is missing",
```

Comment:
This is a comment, let's represent it that way. Can you please change it to something like
journal: undefined, // Can't be migrated because...